holopy/inference/tests/test_interface.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2011-2016, Vinothan N. Manoharan, Thomas G. Dimiduk,
+# Rebecca W. Perry, Jerome Fung, and Ryan McGorty, Anna Wang
```

Comment:
Add your names to the copyright

Reply:
I added my name to all files I have contributed to.